server/views/components/new-site-notice/index.config.js

```diff
@@ -0,0 +1,3 @@
+export const name = 'new-site-notice';
+export const handle = 'new-site-notice';
+export const status = 'benched';
```

Comment:
Not for now, but I wonder if we could create another folder for benched / deprecated components, just for tidiness.